steps/waldo-upload/0.2.0/step.yml

```diff
@@ -0,0 +1,90 @@
+title: Waldo Upload
+summary: |
+  Uploads a build to Waldo.
+description: |
+  Uploads a build to Waldo.
+website: https://github.com/waldoapp/bitrise-step-waldo-upload
+source_code_url: https://github.com/waldoapp/bitrise-step-waldo-upload
+support_url: https://github.com/waldoapp/bitrise-step-waldo-upload/issues
+published_at: 2020-09-21T10:28:59.961629-04:00
+source:
+  git: https://github.com/waldoapp/bitrise-step-waldo-upload.git
+  commit: a1dc3ed369c0fcb3a7d20fa7bb2070dec9f61fba
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
+- xamarin
+- react-native
+- cordova
+- ionic
+- flutter
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: curl
+  apt_get:
+  - name: curl
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- build_path: null
+  opts:
+    description: |
+      A `.app`, `.ipa`, or `.apk` path.
+
+      Typically this path is generated by a previous build Step in your
+      Workflow. For example:
+
+      - the **Xcode build for simulator** Step generates a `.app` at `$BITRISE_APP_DIR_PATH`
+      - the **Xcode Archive & Export for iOS** Step generates a `.ipa` at `$BITRISE_IPA_PATH`
+      - the **Android Build** Step generates a `.apk` at `$BITRISE_APK_PATH`
+    is_required: true
+    summary: A `.app`, `.ipa`, or `.apk` path.
+    title: Build path
+- opts:
+    description: |
+      Each application registered in Waldo is associated with a unique upload
+      token.
+    is_required: true
+    is_sensitive: true
+    title: Waldo upload token
+  upload_token: $WALDO_UPLOAD_TOKEN
```

Comment:
Only env vars exported by bitrise.io, CLI, other steps or shell should be used as default values. Not custom ones.

Reply:
Yeah, that makes sense in this context. I'll remove it.